ioServ.py

```diff
@@ -3,49 +3,35 @@
 from math import floor
 from time import strftime
 
+import guiType
+import rapidjson
```

Comment:
since external libraries can be a pain, there should probably be a check that the user has rapidjson installed. although its not necessary to do now, its something that should be considered in the future.